admin/src/pages/MarketDetails.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from "react";
+import { Form } from "react-bootstrap";
+import { useParams, useNavigate } from "react-router-dom";
+import "@pathofdev/react-tag-input/build/index.css";
+import {
+  fetchMarketById,
+  addVendor,
+  updateMarket,
+} from "../components/Service";
+import FormTextInput from "../components/FormTextInput";
+import FormTextAreaInput from "../components/FormTextAreaInput";
+import MarketDetailsButtons from "../components/MarketDetailsButtons";
+import TimeInput from "../components/TimeInput";
+
+const MarketDetails = () => {
+  const id = useParams().id;
+  const navigate = useNavigate();
+
+  const [success, setSuccess] = useState(false);
+
+  const fetchMarket = async (id: string) => {
+    const response = await fetchMarketById(id);
+    setUpdatedMarket(response);
```

Comment:
Lehet érdemes lenne react-query-t (https://react-query.tanstack.com/) használni itt az adminon is, ahogy tettük a site-on. Beszéljetek @dzsak -kal erről, ő foglalkozott vele a site részen.